build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
Like @kasparsd said, it is all about the purpose of this repository.

Like he said, the goal here is to use the repo as an update channel for future upgrades, so you can suffer the ~250 MiB of the initial download because it's easier/faster when updating. 

My point when I wrote this PR was that 90% of people will not update frequently, and that the other 10% would just `git remote set-branches origin '*'` to get the full repo.